### PR TITLE
Fix number of tag/values computation when the review is skippable

### PR DESCRIPTION
**Pull request by nroggeman-ledger**
## Description

The goal of this PR is to fix https://github.com/LedgerHQ/ledger-secure-sdk/issues/775.
it consists in taking the "skippable" feature into account in the computation of the number of tag/values fitting in a page

## Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Tests
- [ ] Documentation
- [ ] Other (for changes that might not fit in any category)

### CODE REVIEW

File: lib_nbgl/src/nbgl_use_case.c [hunk 1]
```diff
@@ -268,31 +268,36 @@ static void bundleNavStartHome(void);
...
+                              const char                *tagline,
+                              bool                       withSettings,
+                              nbgl_homeAction_t         *homeAction,
+                              nbgl_callback_t            topRightCallback,
+                              nbgl_callback_t            quitCallback);
+static uint8_t useCaseGetNbTagValuesInPage(uint8_t                           nbPairs,
+                                           const nbgl_contentTagValueList_t *tagValueList,
+                                           uint8_t                           startIndex,
+                                           bool                              isSkippable,
+                                           bool *requireSpecificDisplay);
```
Comment (tdejoigny-ledger) [inline lib_nbgl/src/nbgl_use_case.c:300]: too much spaces after `void`

File: lib_nbgl/src/nbgl_use_case.c [hunk 2]
```diff
@@ -1528,7 +1539,8 @@ static uint8_t nbgl_useCaseGetNbPagesForContent(const nbgl_content_t *content,
 
 static uint8_t nbgl_useCaseGetNbPagesForGenericContents(
```
Comment (cedelavergne-ledger) [inline lib_nbgl/src/nbgl_use_case.c:1540]: I thought static function should not start by `nbgl_` :thinking:

File: lib_nbgl/src/nbgl_use_case.c (hunk 1 above)
Comment (nroggeman-ledger) [reply lib_nbgl/src/nbgl_use_case.c:300]: auto-formating, cannot be changed

File: lib_nbgl/src/nbgl_use_case.c (hunk 2 above)
Comment (nroggeman-ledger) [reply lib_nbgl/src/nbgl_use_case.c:1540]: I've not changed it, it was already like this...

Comment (cedelavergne-ledger) [reply lib_nbgl/src/nbgl_use_case.c:1540]: I know :stuck_out_tongue_winking_eye:

**Review by cedelavergne-ledger — APPROVED** [review]

**Pull request merged**